plugins/mail_from.is_resolvable.js

```diff
@@ -24,7 +24,8 @@ exports.load_ini = function () {
 exports.hook_mail = function(next, connection, params) {
     var plugin    = this;
     var mail_from = params[0];
-    var results   = connection.transaction.results;
+    var txn       = connection.transaction;
+    var results   = txn.results;
```

Comment:
Is this just as likely to explode if the transaction doesn't exist? IOW, shouldn't there be a `if (!txn) return;` sandwiched between those two lines?